app/src/main/java/com/example/ide/ui/screen/ChatScreen.kt

```
@@ -1,524 +1,524 @@
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.example.ide.data.model.ChatMessage
+import com.example.ide.ui.viewmodel.MainViewModel
+import kotlinx.coroutines.launch
+import java.text.SimpleDateFormat
+import java.util.*
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ChatScreen(viewModel: MainViewModel) {
```

detekt (1.23.8)

[warning] The function ChatScreen appears to be too complex based on Cognitive Complexity (complexity: 32). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)